persistence/sql/migrations/sql/20230706000000000001_available_aal.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE identities ADD COLUMN available_aal VARCHAR(4) NOT NULL DEFAULT 'aal0';
```

Comment:
Doesn't `NOT NULL DEFAULT ...` do a full table update? Wouldn't it be better to use a nullable column here instead?
Like this every identity would have `available_aal` set to 0 after the migration, and only on update we would set the correct value.

Reply:
Yep, absolutely. However as I understood the people from CRDB, this should not be an issue due to the way migrations work. We still need to verify that though.

Reply:
It's now nullable